packages/venia-concept/__mocks__/@magento/peregrine.js

```diff
@@ -0,0 +1,7 @@
+export const mockRequest = jest.fn();
```

Comment:
I think it's better to put mocks like this inline in your test, or lift them out to test helpers. A manual mock like this will take effect for any future tests in the Venia package that run `jest.mock('@magento/peregrine')`, and this may not be useful for all of them.

Basically, manual mocks are not a good idea until the mocked object's entire behavior is well understood.